frontend/src/components/main/main.module.css

```diff
@@ -0,0 +1,39 @@
+.main_container{
+    margin-top: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+}
+.main__video{
+    position: relative;
+    margin-top: 1rem;
+    max-width: 100%;
+    max-height: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    padding: 10px;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px;
```

Comment:
box shadow could be made lighter, it is bit difficult to differentiate between cards

Reply:
done